### ISSUE TITLE
Upgrade to 2.0.5 issues with db

### ISSUE THREAD

**Issue by stevenvolt** [cuckoosandbox/cuckoo#2034]
Attempted to upgrade but had a migration db issue, I first received an error but was able to resolve it by reviewing a closed issue here.

first issue which was resolved:
cuckoo.common.exceptions.CuckooOperationalError: The path that you specified is not a proper Cuckoo setup. Please point the path to the root of your older Cuckoo setup, i.e., to the directory containing the cuckoo.py script!

Second issue which i'm stuck at.

oisadmin@oisadmin:~$ cuckoo --cwd .cuckoo3 import --symlink /home/oisadmin/.cuckoo
You are importing an existing Cuckoo setup. Please
understand that, depending on the mode taken, if 
you remove the old Cuckoo setup after this import 
you may still lose ALL of your data!

Additionally, database migrations will be performed 
in-place*. You won't be able to use your old Cuckoo 
setup anymore afterwards! However, we'll provide 
you with the option to create a SQL backup beforehand.

TL;DR Cleaning the old setup after the import may
corrupt your new setup: its SQL, MongoDB, and 
ElasticSearch database may be dropped and, in 'symlink'
mode, the analyses removed.

*: Except for sqlite3 databases in combination with
   the import 'copy' approach.

... I've read the above and understand the consequences [y/N]: y
We've identified a Cuckoo Sandbox 2.0.4 installation!
Reading in the old configuration..
  configuration has been migrated to the latest version!

We can make a SQL database backup as follows:
input cmd  => sqlite3 /home/oisadmin/.cuckoo/db/cuckoo.db .dump
output SQL => /home/oisadmin/.cuckoo3/backup.sql
Would you like to make a backup [Y/n]: n
Traceback (most recent call last):
  File "/usr/local/bin/alembic", line 11, in <module>
    load_entry_point('alembic==0.8.8', 'console_scripts', 'alembic')()
  File "/usr/local/lib/python2.7/dist-packages/alembic/config.py", line 479, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/usr/local/lib/python2.7/dist-packages/alembic/config.py", line 473, in main
    self.run_cmd(cfg, options)
  File "/usr/local/lib/python2.7/dist-packages/alembic/config.py", line 456, in run_cmd
    **dict((k, getattr(options, k)) for k in kwarg)
  File "/usr/local/lib/python2.7/dist-packages/alembic/command.py", line 174, in upgrade
    script.run_env()
  File "/usr/local/lib/python2.7/dist-packages/alembic/script/base.py", line 407, in run_env
    util.load_python_file(self.dir, 'env.py')
  File "/usr/local/lib/python2.7/dist-packages/alembic/util/pyfiles.py", line 93, in load_python_file
    module = load_module_py(module_id, path)
  File "/usr/local/lib/python2.7/dist-packages/alembic/util/compat.py", line 79, in load_module_py
    mod = imp.load_source(module_id, path, fp)
  File "./env.py", line 59, in <module>
    run_migrations_online()
  File "./env.py", line 47, in run_migrations_online
    connection = engine.connect()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 2018, in connect
    return self._connection_cls(self, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 72, in __init__
    if connection is not None else engine.raw_connection()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 2104, in raw_connection
    self.pool.unique_connection, _connection)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 2078, in _wrap_pool_connect
    e, dialect, self)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 1405, in _handle_dbapi_exception_noconnection
    exc_info
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/util/compat.py", line 199, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/base.py", line 2074, in _wrap_pool_connect
    return fn()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 318, in unique_connection
    return _ConnectionFairy._checkout(self)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 713, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 480, in checkout
    rec = pool._do_get()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 1151, in _do_get
    return self._create_connection()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 323, in _create_connection
    return _ConnectionRecord(self)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 449, in __init__
    self.connection = self.__connect()
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/pool.py", line 607, in __connect
    connection = self.__pool._invoke_creator(self)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/strategies.py", line 97, in connect
    return dialect.connect(*cargs, **cparams)
  File "/usr/local/lib/python2.7/dist-packages/sqlalchemy/engine/default.py", line 385, in connect
    return self.dbapi.connect(*cargs, **cparams)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Traceback (most recent call last):
  File "/usr/local/bin/cuckoo", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/main.py", line 635, in import_
    import_cuckoo(ctx.parent.user, mode, path)
  File "/usr/local/lib/python2.7/dist-packages/cuckoo/apps/import_.py", line 177, in import_cuckoo
    "Error migrating your old Cuckoo database!"
cuckoo.common.exceptions.CuckooOperationalError: Error migrating your old Cuckoo database!

**Comment by jbremer** [comment]
Sorry. You can only `cuckoo import` a legacy setup. For Cuckoo 2.0 and higher setups you can simply `pip install -U cuckoo` and it'll do the rest (semi-)automatically. We should probably make that a bit more clear, @RicoVZ.

**Comment by jbremer** [comment]
Closing issue as resolved.
